layouts/shortcodes/keps-data.html

```diff
@@ -0,0 +1,62 @@
+<script src="/js/keps/kep_search.js"></script>
+<div>
+  <dir class="d-flex">
+    <div class="col">
+      <div class="input-group mt-2 col-md-12">
+        <input type="text" name="search" id="keps_search" class="form-control" placeholder="Search KEP(s) by keyword" />
+      </div>
+    </div>
+    <div class="col ml-auto">
+      <div class="input-group mx-auto mt-2 col-md-12" style="">
+        <label for="keps" class="pr-3">Choose SIG:</label>
+        <select class="custom-select" name="sig group" id="sig_group_select">
+          <option selected="selected" value="all">Choose...</option>
+          {{ range $data_sig := .Site.Data.sigs.sigs.sigs }}
+          <option value="{{$data_sig.dir}}">
+            {{ printf "%s" $data_sig.dir }}
+          </option>
+          {{ end }}
+        </select>
+      </div>
+    </div>
+  </dir>
+  <table id="keps_table" class="table table-bordered table-hover mt-2 col-md-12">
+    <thead>
+      <tr>
+        <th scope="col">KEP Number</th>
+        <th scope="col">Title</th>
+        <th scope="col">SIG</th>
+        <th scope="col">Author</th>
+      </tr>
+    </thead>
+    <tbody>
+      {{ range $index, $data := sort $.Site.Data.keps.keps_data.keps "kep_number" }}
+      <tr>
+        <td>
+          <a href="https://kep.k8s.io/{{ $data.kep_number }}">
+            {{ printf "%s" $data.kep_number }}
+          </a>
+        </td>
+        <td>
+          <a href="https://github.com/kubernetes/enhancements/tree/master/keps/{{$data.owning_sig}}/{{$data.name}}#summary">
+            {{ printf "%s" $data.title }}
+          </a>
+        </td>
+        <td>
+          <a href=" https://github.com/kubernetes/community/blob/master/{{ $data.owning_sig }}/README.md">
+            {{ printf "%s" $data.owning_sig }}
+          </a>
+        </td>
+        <td>
+          {{ range $index, $author := $data.authors }}
+            {{- if $index -}}, {{ end -}}
+            <a href="https://github.com/{{ substr $author 1}}">
+              {{ $author}}
```

Comment:
This normalizes the hyperlink and GitHub username (and makes styling a bit easier):
```suggestion
            <a href="https://github.com/{{ trim $author "@" }}">
              <span class="github-user">@{{ trim $author "@" }}</span>
```